apps/api/v2/src/modules/verified-resources/outputs/verified-email.output.ts

```diff
@@ -0,0 +1,73 @@
+import { ScheduleOutput } from "@/ee/schedules/schedules_2024_04_15/outputs/schedule.output";
+import { ApiProperty } from "@nestjs/swagger";
+import { Expose, Type } from "class-transformer";
+import { IsArray, IsEnum, IsNotEmptyObject, IsNumber, IsString, ValidateNested } from "class-validator";
+
+import { ERROR_STATUS, SUCCESS_STATUS } from "@calcom/platform-constants";
+
+export class VerifiedEmailOutputData {
+  @Expose()
+  @IsNumber()
+  @ApiProperty({
+    description: "The unique identifier for the verified email.",
+    example: 789,
+  })
+  id!: number;
+
+  @Expose()
+  @IsString()
+  @ApiProperty({
+    description: "The verified email address.",
+    example: "user@example.com",
+    format: "email",
+  })
+  email!: string;
+
+  @Expose()
+  @IsNumber()
+  @ApiProperty({
+    description: "The ID of the associated user, if applicable.",
```

Comment:
Shall we have 2 classes `UserVerifiedEmailOutputData` and `TeamVerifiedEmailOutputData` so we don't have optional props.